finality-aleph/src/finalization.rs

```diff
@@ -216,25 +213,24 @@ impl<
     }
 
     pub(crate) async fn start_aggregation(&mut self, hash: B::Hash) {
+        debug!(target: "afa", "Started aggregation for block hash {:?}", hash);
         if !self.started_hashes.insert(hash) {
             return;
         }
         self.hash_queue.push_back(hash);
-        self.rmc.start_rmc(SignableHash { hash }).await
+        self.rmc.start_rmc(SignableHash { hash }).await;
+        debug!(target: "afa", "Started rmc for block hash {:?}", hash);
```

Comment:
Actually I have added the second one, because I saw `.await` and wanted to catch whether something bad happens there, but you are right, enough to have one debug here.